uaclient.conf

```diff
@@ -15,5 +15,4 @@ ua_config:
   https_proxy: null
```

Comment:
I opted not to add it here just because most people won't care about this setting in particular, so it didnt seem worth adding to the default conf.

Thats not a strong opinion at all though, if you think I should add it, I will!

Reply:
Makes sense - agreed